src/oemof/solph/flows/_flow.py

```diff
@@ -172,6 +168,15 @@ def __init__(self, **kwargs):
             "positive_gradient": {"ub": None},
             "negative_gradient": {"ub": None},
         }
+        need_nominal_value = [
+            "fix",
+            "full_load_time_max",
+            "full_load_time_min",
+            "max",
+            "min",
+            "summed_max",
+            "summed_min",
+        ]
```

Comment:
You could add a hint here that `summed_min/max` is deprecated, as a reminder, but this is just an idea.